src/EditorFeatures/CSharpTest/EditAndContinue/ActiveStatementTests.cs

```diff
@@ -9065,5 +9065,55 @@ public static void F()
         }
 
         #endregion
+
+        #region C# 8.0
+
+        [Fact]
+        public void UpdateAroundActiveStatement_SwitchExpression_PositionalPattern()
```

Comment:
>public [](start = 8, length = 6)

This test isn't useful here. We need to test that the variables are mapped correctly. See LocalSlotMappingTests